src/formio/components/Select.stories.js

```diff
@@ -131,3 +132,57 @@ export const MultilineOptions = {
     },
   },
 };
+
+export const WithIntegerValues = {
```

Comment:
can't we write a unit test for this instead like in `src/jstests/formio/components/textfield.spec.js`? Chromatic snapshots cost money, and checking the type of values is really not an interaction test.

Reply:
Replaced it with a regular test